public/assets/css/style.css

```diff
@@ -16,9 +16,6 @@ a {
 body {
     overflow-x: hidden;
     margin: 0;
-    display: flex;
-    flex-direction: column;
-    justify-content: space-between;
```

Comment:
@mchlltt I tested this pretty extensively. It looked like the flex stuff was only necessary for the options.